package.json

```diff
@@ -18,9 +19,11 @@
   },
   "homepage": "https://github.com/pelias/query#readme",
   "dependencies": {
-    "check-types": "^7.0.0"
+    "check-types": "^7.0.0",
+    "jshint": "^2.9.2"
   },
   "devDependencies": {
+    "jshint": "^2.5.6",
```

Comment:
if you run `npm install -D precommit-hook && npm install`, precommit-hook will automatically generate precommit hooks that run the unit tests and lint before every git commit, to make sure they are passing. We use it in most of our other repos so we should do it here too.